src/services/converters/ImageMediaConverter.ts

```diff
@@ -41,14 +41,22 @@ export default class ImageMediaConverter extends MediaConverter {
     }
 
     const url = `${process.env.BASE_URL}${this.path}`;
-    const request = fetch(url, {
-      method: "POST",
-      body,
-    });
-    const response = await (await request).json();
-    if (!this.isMediaResponse(response)) {
-      throw Error("Invalid Response");
+
+    try {
+      const request = fetch(url, {
+        method: "POST",
+        body,
+      });
+
+      const response = await (await request).json();
+
+      if (!this.isMediaResponse(response)) {
+        throw Error(response.toString());
+      }
+
+      return this.responseToMediaFileArray(response, "image/t3x");
+    } catch (exception) {
+      throw Error((exception as Error).message);
```

Comment:
Why use the `try .. catch` block if we're rethrowing any exception we catch anyway?

Reply:
If for some reason the backend is down, it causes the thing to blow up. I probably should have done it for FonrtMediaConverter as well.

Reply:
Weird, but OK, justified